### PR TITLE
CR-1122352: Updated aie_trace specific config parameters

### PR DESCRIPTION
**Pull request by nishraptor**
Created new options for the periodic aie_trace offload to decouple form the continuous_trace option. Reverted default for aie_trace_metrics to "functions".

### CODE REVIEW

**Comment by stsoe** [comment]
ok to test

**Review by stsoe — APPROVED** [review]

**Review by jvillarre — APPROVED** [review]

**Comment by nishraptor** [comment]
@pgschuey

**Comment by dayeh-xilinx** [comment]
retest this please.

**Review by pgschuey — APPROVED** [review]

**Comment by kbhardwa-xilinx** [comment]
retest this please.

**Comment by pgschuey** [comment]
retest this please

File: src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp [hunk 1]
```diff
@@ -89,9 +89,9 @@ namespace xdp {
```
Comment (rozumx) [inline src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.cpp:89]: Copyright date needs to be updated (line 2).

File: src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h [hunk 2]
```diff
@@ -82,6 +82,7 @@ namespace xdp {
```
Comment (rozumx) [inline src/runtime_src/xdp/profile/plugin/aie_trace/aie_trace_plugin.h:82]: Copyright date needs to be updated (line 2).

File: src/runtime_src/xdp/profile/writer/aie_trace/aie_trace_writer.cpp [hunk 3]
```diff
@@ -107,7 +107,7 @@ std::cout << " AIETraceWriter::writeTraceEvents : dataBuffer " << dataBuffer <<
   {
```
Comment (rozumx) [inline src/runtime_src/xdp/profile/writer/aie_trace/aie_trace_writer.cpp:107]: Copyright date needs to be updated (line 2).

**Review by rozumx — APPROVED** [review]
Code changes looks good though the copyright dates need to be updated.

**Pull request merged**